### DISCUSSION TITLE
[자료구조 3일차] 트리

### DISCUSSION THREAD

**Discussion by seongeun42** [Shortudy-10th/tech-for-developer#32] [오늘의 주제]
트리 관련 질문 2인조 1개, 3인조 2개씩 총 5개

**Comment by sikutisa** [comment]
B트리와 B+트리의 구조와 그 차이점에 대해 설명해주세요

**Comment by sikutisa** [comment]
* 레드-블랙트리에 대해 설명해주세요.
   * 심화질문(이건 필수로 할 필요 없을 듯): 레드-블랙트리에 새로운 데이터를 삽입할 때, 내부 동작 방식에 대해 설명해주세요.

**Comment by watty-exclamationMark** [comment]
AVL트리에 대해 해당 트리의 삽입, 검색, 삭제의 시간 복잡도를 포함하여 설명해주세요.

**Comment by zini9188** [comment]
트리 순회를 위해 사용하는 방법에는 무엇이 있는지 설명해주세요

**Comment by watty-exclamationMark** [comment]
![tree](https://github.com/Shortudy-10th/tech-for-developer/assets/70586307/8e295580-04d5-47f9-b5c0-1b886cfacb5d)
다음은 제가 임의로 만든 트리 이미집니다...
해당 트리의 높이와 차수를 말해주세요.

**Comment by kjc6735** [comment]
BST(Binary Search Tree)와 Binary Tree에 대해 설명해주세요.
